syntax = "proto2";

message ImageMsg {
  optional string original_filename = 1;
  optional int32 width = 2;
  optional int32 height = 3;
  optional bytes image_data = 4;
}
